public/images/activity/button.svg

```diff
@@ -0,0 +1,12 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="53" height="29" viewBox="0 0 53 29">
+  <g id="组_2999" data-name="组 2999" transform="translate(-1053 -10)">
+    <g id="矩形_2832" data-name="矩形 2832" transform="translate(1053 10)" fill="none" stroke="#37fbf6" stroke-width="1">
+      <rect width="53" height="29" rx="14.5" stroke="none"/>
+      <rect x="0.5" y="0.5" width="52" height="28" rx="14" fill="none"/>
+    </g>
+    <g id="组_2998" data-name="组 2998" transform="translate(-0.541 -0.278)">
+      <path id="路径_3912" data-name="路径 3912" d="M2460.939-1685.97l6.222,6.222-6.222,6.222" transform="translate(-1378.701 1704.248)" fill="none" stroke="#36f9f5" stroke-linecap="round" stroke-width="2"/>
+      <path id="路径_3913" data-name="路径 3913" d="M2460.939-1685.97l6.222,6.222-6.222,6.222" transform="translate(-1388.398 1704.248)" fill="none" stroke="#36f9f5" stroke-linecap="round" stroke-width="2"/>
+    </g>
+  </g>
+</svg>
```

Comment:
SVG image should be compressed before it's committed into the codebase.
Use https://jakearchibald.github.io/svgomg/